qemu/tests/cfg/qemu_disk_img.cfg

```diff
@@ -2,7 +2,7 @@
 # For windows guest, you can download md5sum.exe from http://www.brothersoft.com/md5sum-81781.html,
 # then add parents dir to $PATH or config md5sum in share/cfg/guest-os/Windows.cfg;
 - qemu_disk_img:
-    no qed raw vmdk
+    no raw vmdk
     virt_test_type = qemu
```

Comment:
It looks like that you want to support qed format for this case.

Reply:
'qed' is compiled out of qemu package, so I think there should be no 'qed' any more. Like 'abc', 'hisd', ... which formats are not supported.

Reply:
How did you know that? Could you share the bug or documentation?

Reply:
The info is from Kevin, you could check the description from the related bug.

Reply:
The description is not suitable.
The reason why QEMU upstream develop QED is to create a better perfoming image format by removing some features compared to qcow2(such as refcount table). It is indeed a official image format. But for downstream qemu, i am not sure. And even for upstream qemu, the development of QED has been abandoned and the improvements have been merged back into qcow2.

Reply:
An other fact is that we could not create qed/vdi image file in latest qemu. It prompts the format is not supported.

[root@lenovo-sr630-02 ~]# qemu-img create -f qed test.img 1G
qemu-img: test.img: Unknown file format 'qed'
[root@lenovo-sr630-02 ~]# qemu-img create -f vdi test.img 1G
qemu-img: test.img: Unknown file format 'vdi'

Reply:
I knew that. But your commit message is not suitable to drop a feature in tp-qemu. As the repo is for upstream testing. If all your changes are only for downstream qemu, the correct way is remove the case in internal test loops.